### PR TITLE
Toolbar menu

### PR DESCRIPTION
**Pull request by ARYA-BHARDWAJ**
This PR is regarding issue no #733. Now  "View" -> "Toolbar" menu item is active.

### CODE REVIEW

**Comment by ARYA-BHARDWAJ** [comment]
@ice0  please have a look

**Comment by ice0** [comment]
Looks good to me.

@morevnaproject should the command show/hide the toolbar for all windows or only for the selected one?

**Comment by ankit-kumar-dwivedi** [comment]
If you ask me I think it's more of like preference so if I hide, it should
not be visible in other windows as well.
However as every window have it's own toolbar so I'm not sure if this is
what is expected.
If we go with 1st option then we should not have separate toolbar for every
window.

**Comment by morevnaproject** [comment]
Reviewing now! ^__^

File: synfig-studio/src/gui/mainwindow.cpp [hunk 1]
```diff
@@ -239,6 +243,14 @@ MainWindow::toggle_show_menubar()
 		menubar->hide();
 }
 
+void
+MainWindow::toggle_show_toolbar()
+{
+	App::enable_mainwin_toolbar = !App::enable_mainwin_toolbar;
+	if (etl::loose_handle<CanvasView> canvas_view = App::get_selected_canvas_view())
+		canvas_view->toogle_displaybar();
```
Comment (morevnaproject) [inline synfig-studio/src/gui/mainwindow.cpp:251]: I suggest to rename `CanvasView::toogle_displaybar()` to `CanvasView::toggle_show_toolbar()`
(to make it correspond with `MainWindow::toggle_show_toolbar()` above).

**Comment by ARYA-BHARDWAJ** [comment]
@morevnaproject please have a look now.

**Comment by morevnaproject** [comment]
@ARYA-BHARDWAJ  Looks good to me now! Can you please rebase your branch on top of current master, so I can make a clean merge?

**Comment by ARYA-BHARDWAJ** [comment]
@morevnaproject I have rebased  the branch on top of current master.

**Comment by morevnaproject** [comment]
@ARYA-BHARDWAJ Hmm... I see it is still not rebased yet. Please make sure that you fetched the latest master commits. You can do this with following commands (I assume that you have main synfig repo as "upstream" remote):
```
git fetch upstream
git rebase upstream/master
git push --force
```

**Comment by ARYA-BHARDWAJ** [comment]
@morevnaproject  sorry my bad I forget to push. I hope I have done it correctly

**Comment by morevnaproject** [comment]
Merged now. Thank you! ^__^

**Pull request merged**